README.md

```diff
@@ -257,7 +257,14 @@ types are hashable as well, so you can construct stranger composite types
       StringIntegerMap(brian => 28, ian => 15, robey => 5000) => 0.2,
       StringIntegerMap(brian => 30, ian => 21, robey => 35) => 0.9)
 
+#### Choice Types ####
 
+Choice types represent alternatives - values that can have one of some set of values.
+
+    >>> C = Choice("IntOrString", [Integer, String])
```

Comment:
I'm not sure using the `Choice(name, alternatives)` call form is useful in the example; i'd stick to just `Choice([Integer, String])` to highlight the relevant point.